addons/l10n_id_pos/models/pos_order.py

```diff
@@ -0,0 +1,8 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    l10n_id_qris_transactions = fields.Many2many('l10n_id.qris.transaction')
```

Comment:
```suggestion
    l10n_id_qris_transaction_ids = fields.Many2many('l10n_id.qris.transaction')
```

normally x2many fields are named with suffix `_ids`.